### PR TITLE
added authenticateByInMemoryPublicKey method to NMSSHSession

### PR DESCRIPTION
**Pull request by zenwheel**

### CODE REVIEW

**Comment by Lejdborg** [comment]
Looks good - just fix the identation so four spaces is used instead of tab.

**Comment by zenwheel** [comment]
Oops, updated the pull request with spaces!

File: NMSSH/NMSSHSession.m [hunk 1]
```diff
@@ -439,6 +439,37 @@ - (BOOL)authenticateByPublicKey:(NSString *)publicKey
     return self.isAuthorized;
 }
 
+- (BOOL)authenticateByInMemoryPublicKey:(NSString *)publicKey
+                     privateKey:(NSString *)privateKey
```
Comment (Lejdborg) [inline NMSSH/NMSSHSession.m]: Final nitpicking: Align `:` on line 443 and 444 with the one on 442.

**Comment by zenwheel** [comment]
updated pull request to fix parameter alignment

**Comment by Lejdborg** [comment]
Thanks!

**Pull request merged**